lib/imgix/client.rb

```diff
@@ -75,6 +75,11 @@ def validate_hosts!
       unless @hosts.length > 0
         raise ArgumentError, "The :host or :hosts option must be specified"
       end
+      @hosts.each do |host|
+        unless host.match(DOMAIN_REGEX) != nil
```

Comment:
This double-negative seems unnecessarily confusing. Can we do `if host.match(DOMAIN_REGEX) == nil` instead?